src/main/webapp/index.jsp

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>canvas_courses_report</title>
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="description" content="">
+<meta name="author" content="">
```

Comment:
Needed? Might consider making this is a standard if to ("UM ITS Teaching and Learning"